CMakeLists.txt

```diff
@@ -292,10 +292,6 @@ if(WIN32)
 endif(WIN32)
 # end editline stuff
 
-if( NOT CPP_STANDARD )
-  set( CPP_STANDARD "-std=c++11" )
```

Comment:
Why not simply change it to c++14 rather than removing it?

Reply:
I believe the lines near the top of this file do that for us. That way, it is all in 1 area, with no extra modifications like the ones below this one. It "does the right thing" on the right platform.